src/types/index.ts

```diff
@@ -211,7 +215,7 @@ export type MasterEventHandler<T extends BaseTestplane> = {
     (event: Events["WARNING"], callback: () => void): T;
     (event: Events["ERROR"], callback: (err: Error) => void): T;
 
-    (event: Events["UPDATE_REFERENCE"], callback: (data: { state: string; refImg: ImageInfo }) => void): T;
+    (event: Events["UPDATE_REFERENCE"], callback: (data: { state: string; refImg: RefImageInfo }) => void): T;
```

Comment:
Actually, `refImg` here still could be  `ImageInfo` type, but only if its emitted from the outside (from the html-reporter, for example)